src/Components/Artwork/Details.tsx

```diff
@@ -163,13 +163,6 @@ const SaleMessage: React.FC<DetailsProps> = ({
     return <>{highestBid_display || openingBid_display || ""}</>
   }
 
-  if (sale_message?.toLowerCase() === "contact for price") {
-    return <>Price on request</>
-  }
```

Comment:
The removal of this condition relates to the work done in:
- https://github.com/artsy/gravity/pull/17851
- https://github.com/artsy/metaphysics/pull/5816

We're handling this logic now in Gravity, so if `Price on request` is needed here, it will come like this from the query and the client doesn't have to worry about it.